cocli/cmd/common.go

```diff
@@ -0,0 +1,85 @@
+// Copyright 2021 Contributors to the Veraison project.
+// SPDX-License-Identifier: Apache-2.0
+
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/afero"
+	"github.com/veraison/corim/comid"
+	"github.com/veraison/swid"
+)
+
+func filesList(files, dirs []string, ext string) []string {
+	var l []string
+
+	for _, file := range files {
+		if _, err := fs.Stat(file); err == nil {
+			l = append(l, file)
+		}
+	}
+
+	for _, dir := range dirs {
+		filesInfo, err := afero.ReadDir(fs, dir)
+		if err != nil {
+			continue
+		}
+
+		for _, fileInfo := range filesInfo {
+			if !fileInfo.IsDir() && filepath.Ext(fileInfo.Name()) == ext {
+				l = append(l, filepath.Join(dir, fileInfo.Name()))
+			}
+		}
+	}
+
+	return l
+}
+
+type FromCBORLoader interface {
+	FromCBOR([]byte) error
+}
+
+func printJSONFromCBOR(fcl FromCBORLoader, cbor []byte, heading string) error {
```

Comment:
I was wondering if these common functions can be made available to a package outside of cocli, in case if it finds a value during logging or debugging?

Reply:
will be dealt with in #11